content/calico/stars_policy_demo/default_policy.md

```diff
@@ -23,7 +23,7 @@ To open the Management UI, retrieve the DNS name of the Management UI using:
 kubectl get svc -o wide -n management-ui
 ```
 
-Copy the `EXTERNAL-IP` from the output, and paste into a browser.
+Copy the **EXTERNAL-IP** from the output, and paste into a browser.
```

Comment:
I would add this to the text to make it clear to grab ELB endpoint

The EXTERNAL-IP column contains a value that ends with "elb.amazonaws.com” - the full value is the DNS address.

Reply:
added